.github/workflows/canary-integration-test.yml

```diff
@@ -1398,6 +1398,71 @@ jobs:
         run: |
           timeout 60 sh -c 'until [ "$(kubectl get cm -n rook-ceph rook-ceph-csi-mapping-config  -o jsonpath='{.data.csi-mapping-config-json}' | grep -c "rook-ceph-secondary")" -eq 1 ]; do echo "waiting for rook-ceph-csi-mapping-config to be created with cluster ID mappings" && sleep 1; done'
 
+      - name: replace operator and cluster images for rados namespace mirroring
+        run: |
+          kubectl -n rook-ceph patch deployment rook-ceph-operator --type='json' -p='[{"op": "replace", "path": "/spec/template/spec/containers/0/image", "value": "quay.io/paarora/rook-ceph:v614"}]'
+          kubectl -n rook-ceph patch cephcluster my-cluster --type='merge' -p '{"spec": {"cephVersion": {"image": "quay.io/paarora/rook-ceph:v609"}}}'
```

Comment:
Instead of modifying to a private image like this, just make the changes in this PR that will change the base image in the operator, and the ceph cluster image directly. Then this PR will test the intended changes. You can rebase it periodically to confirm it is still working, but then we will just wait for a while to merge the test until v20 is released and we are ready to make those changes in Rook.

Reply:
Actually, for testing, I have already done  in the live cluster,
okay will keep on hold till the ceph release is out and run this periodically.